lib/link_text.dart

```diff
@@ -22,13 +22,19 @@ class LinkText extends StatefulWidget {
   /// Determines how the text is aligned.
   final TextAlign textAlign;
 
+  /// if true it will cut off all visible links after '?'.
+  /// This is only for better readability. When executing the url
+  /// the link with all params stays the same.
+  final bool trimParams;
```

Comment:
I think `shouldTrimParams` name might be a bit better

Reply:
Good point :-)